solr/webapp/web/js/angular/controllers/cloud.js

```diff
@@ -971,7 +974,10 @@ solrAdminApp.directive('graph', function(Constants) {
                 var w = element.width(),
                     h = leafCount * 20;
 
-                var tree = d3.layout.tree().size([h, w - 400]);
+                // Calculate roughly the width of host name to align the graph
+                var hostnameWidth = graphData.children.length > 0 ?
+                  graphData.children[0].children[0].children[0].name.length * 7.5 : 400;
```

Comment:
`7.5` must have been some fun trial and error!

Reply:
Will test with a few other widths I suppose. I'm trying a formula with substracting legend box width and then calculating hostname width using chars * `5.5` :)